Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -143,3 +268,29 @@ extension ProjectPamphletViewController: ProjectNavBarViewControllerDelegate {
     self.contentController.tableView.scrollToTop()
   }
 }
+
+// MARK: - Styles
+
+private var backThisProjectButtonStyle = { (button: UIButton) -> UIButton in
+  button
+    |> greenButtonStyle
+    |> roundedStyle(cornerRadius: 12)
+    |> UIButton.lens.layer.borderWidth .~ 0
+    |> UIButton.lens.titleEdgeInsets .~ .init(topBottom: Styles.grid(1), leftRight: Styles.grid(2))
+    |> UIButton.lens.title(for: .normal) %~ { _ in
+      return Strings.project_back_button()
+  }
+}
+
+private var backThisProjectButtonTitleLabelStyle = { (titleLabel: UILabel?) -> UILabel? in
+  _ = titleLabel
+    ?|> \.font .~ UIFont.ksr_headline()
+    ?|> \.numberOfLines .~ Int(0)
```

Comment:
Isn't `0` inferred to `Int` by default?

Reply:
Oh I was trying to help the compiler here as much as possible 😭